src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -62,6 +63,26 @@ const HeaderUserMenu = ({ user }) => {
             ''
           )}
         </div>
+        {user.sms.description ? (
+          <div className="user-plan--container">
+            <div className="user-plan--type">
+              <p><strong className={user.sms.remainingCredits < 0 ? 'red' : ''}>US$ {' '}
```

Comment:
maybe its better a function to get the className to have the html more clean. what do you think @cbernat

Reply:
I agree, would you mind change this into a function @lucianaDilizio please?